sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 5.9.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.8.1 (2023-03-07)
 
 ### Bugs Fixed
 
-### Other Changes
+- Fix null reference exception when using the `EventProcessorClient`.
```

Comment:
```suggestion
- Fix null reference exception when using the an event processor.
```

Can we reflect the `EventProcessorClient` one over to the change log for the processor package?

Reply:
ah woops.

Reply:
Hmm I suppose we don't need this change entry in the main package.